.gitignore

```diff
@@ -8,6 +8,9 @@
 #notes
 todo.txt
 
+#db
+/prisma/migrations
```

Comment:
I think this shouldn't be ignored, it should be commited as it contains a history of migrations (changes in db schema). Please confirm with Rafał Ch.